WEB/Src/Web/Web.Shared.Net/ApplicationInsightsHttpModule.cs

```diff
@@ -82,15 +98,34 @@ public ApplicationInsightsHttpModule()
         /// <summary>
         /// Initializes module for a given application.
         /// </summary>
-        /// <param name="context">HttpApplication instance.</param>
+        /// <param name="application">HttpApplication instance.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1062:Validate arguments of public methods", MessageId = "0", Justification = "Context cannot be null")]
-        public void Init(HttpApplication context)
+        public void Init(HttpApplication application)
```

Comment:
I agree with this change, but it might break customers who use named parameters.